test(devtools): write some e2e tests for the transfer state tab

AleksanderBodurri opened the pull request:
Tests validate the current behaviour of the transfer state feature:

- Tab can be made visible by enabling in the settings menu
- Transfer state component displays keys total and size
- Validates the existence of the mock content present in https://github.com/angular/angular/tree/main/devtools/src/app/transfer-state.ts

crisbeto commented:
This PR was merged into the repository by commit 3a1a6cc20deb10fe790a6d8137c5b7b95c3ddf17.

The changes were merged into the following branches: main

AleksanderBodurri committed:
test(devtools): write some e2e tests for the transfer state tab

Tests validate the current behaviour of the transfer state feature:

- Tab can be made visible by enabling in the settings menu
- Transfer state component displays keys total and size
- Validates the existence of the mock content present in https://github.com/angular/angular/tree/main/devtools/src/app/transfer-state.ts

AleksanderBodurri commented:
It looks like this broke the first time because it conflicted with changes made in https://github.com/angular/angular/issues/62851
